acrobat/blocks/app-banner/app-banner.js

```diff
@@ -0,0 +1,128 @@
+import { createTag } from "../../scripts/miloUtils.js";
+
+var link = '';
+var mobileOS = getMobileOperatingSystem();
+
+
+export default async function init(el) {
+  //Exit if not mobile/tablet 
+  if (mobileOS == 'unknown') {
+    el.innerHTML = '';
+    return;
+  }
+
+  createTag.then((createTag) => {
+    const bannerPlaceHolder = createTag('div', { class: 'placeholder-banner' }, '');
+    document.body.prepend(bannerPlaceHolder);
+    const children = el.querySelectorAll(':scope > div');
+    if (mobileOS === 'Android') {
+      link = children[4].textContent.trim();
+    } else if (mobileOS === 'iOS') {
+      link = children[7].textContent.trim();
+    }
+    const appBanner = createTag('div', { class: 'app-banner' }, '');
+    //Close banner button on the left, also the icon
+    const closeBtn = createTag('div', { class: 'app-banner-close', role: 'text', 'aria-label': 'Close banner' }, '×');
+    closeBtn.addEventListener('click', closeBanner);
+    const appBannerContent = createTag('div', { class: 'app-banner-content' }, '');
+    appBannerContent.append(closeBtn, getDecoratedBannerLeft(createTag, children),
+      getDecoratedAppDetails(createTag, children), getDecoratedBannerRight(createTag, children));
+    appBanner.append(appBannerContent);
+    el.innerHTML = '';
+    appBanner.style.display = 'block';
+    bannerPlaceHolder.remove();
+    document.body.classList.add('has-app-banner');
+    document.body.prepend(appBanner);
+  });
+}
+
+function openLink() {
+  window.open(link, "_blank");
+  closeBanner();
+}
+
+function closeBanner() {
+  const appBanner = document.querySelector('.app-banner');
+  document.body.classList.remove('has-app-banner');
+  appBanner.remove();
+}
+
+function getDecoratedBannerRight(createTag, children) {
+  const openText = children[3].textContent.trim();
+  const appBannerRight = createTag('div', { class: 'app-banner-right' }, '');
+  const openButton = createTag('div', { class: 'app-banner-button', role: 'text', 'aria-label': openText }, openText);
```

Comment:
I would use `button` tag instead of div if we need a button

Reply:
Bringing in the existing design from branch.io. Need to see as it may impact the style if I add button ?

Reply:
<img width="381" alt="image" src="https://github.com/adobecom/dc/assets/116916138/b333d300-fc49-45ff-a8bd-cb0ce340959b">
It grays out the button which needs to be transparent.

Reply:
You should be able to set button background to transparent.

Reply:
Done